src/components/tonies/TonieCard.tsx

```diff
@@ -1,300 +1,469 @@
-import React, { useState, useEffect } from 'react';
-import { useTranslation } from 'react-i18next';
-import { Card, Button, Input, Popover, message, Modal } from 'antd';
-import { InfoCircleOutlined, PlayCircleOutlined, CloudSyncOutlined, RetweetOutlined, DownloadOutlined, EditOutlined, SaveOutlined, CloseOutlined, StarFilled } from '@ant-design/icons';
+import {
```

Comment:
There seem to be many changed lines due to a different auto format setting (There is nothing enforced currently).
What are you using?